src/lib/QtWidgets/QBoxLayout.ts

```diff
@@ -38,6 +41,7 @@ export class QBoxLayout extends NodeLayout {
     }
     insertLayout(index: number, layout: NodeLayout, stretch = 0): void {
         this.native.insertLayout(index, layout.native, stretch);
```

Comment:
seems like there can be multiple layouts. then i think the memeber field can be a `Set` instead.
so you can have a field like `childLayout?:Set<NodeLayout>`

Reply:
Sounds good, i'll try it later when i'm home. I'm not sure how to work with Sets, can i just initialize it with `this.childLayouts = new Set()` and then add with `childLayouts.add(layout)`? Should i also check it with `childLayouts.has(layout)` before adding?(it already logs when u try to add the same layout, not sure where it comes from)

Reply:
>Sounds good, i'll try it later when i'm home. I'm not sure how to work with Sets, can i just initialize it with this.childLayouts = new Set() and then add with childLayouts.add(layout)
Yes thats pretty much it 👍 

No you dont need to check it before adding. Its on the user.